web/packages/shared/services/loc/loc.ts

```diff
@@ -83,3 +83,19 @@ export function displayDateTime(date: Date) {
 export function dateToUtc(date: Date) {
   return new Date(date.getTime() + date.getTimezoneOffset() * 60 * 1000);
 }
+
+export function dateTimeShortFormat(date: Date) {
```

Comment:
It'd be nice to put an example in a JSDoc of how the formatted date is going to look like, say for en-US at least. And maybe mention that it's going to use browser and system settings to determine the actual format that's going to be used.